### PR TITLE
sqlmap 1.0

### PR DESCRIPTION
**Pull request by youtux**
### All Submissions:

- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### CODE REVIEW

File: Library/Formula/sqlmap.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class Sqlmap < Formula
   desc "Penetration testing for SQL injection and database servers"
   homepage "http://sqlmap.org"
-  url "https://github.com/sqlmapproject/sqlmap/archive/0.9.tar.gz"
-  sha256 "77b2b0ac7844cbb045baa8d477ca11a9f3ca736b7e0b340787c8a181a2fc718f"
+  url "https://github.com/sqlmapproject/sqlmap/archive/1.0.tar.gz"
+  sha256 "6818c74d6bd5525879beedc9baaef404534cd5b8c0f671476367783fb3ef64a6"
   head "https://github.com/sqlmapproject/sqlmap.git"
```
Comment (mathiasbynens) [inline Library/Formula/sqlmap.rb]: I think `head` can be removed as well

Comment (youtux) [inline Library/Formula/sqlmap.rb]: Why is that?

Comment (mathiasbynens) [inline Library/Formula/sqlmap.rb]: It’s only needed when building from `HEAD` using `--HEAD` which your patch removed support for.

Comment (mathiasbynens) [inline Library/Formula/sqlmap.rb]: Ah, my bad, I misread the deleted lines in the patch. Carry on :)

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/0673876278b1f4b7e5ae4c0fe4a1e1c353d26f1d. Thank you for another contribution to Homebrew @youtux! :smiley:

**Pull request closed without merging**